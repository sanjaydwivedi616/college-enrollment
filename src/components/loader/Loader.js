import ReactDOM from 'react-dom';

const Loader = () => {

   // const spiner = document.getElementById("spiner");
    
    return (
           // ReactDOM.createPortal(<div className='loader'></div>, spiner)      
           <p>Loading.....</p> 
    )
}

export default Loader;