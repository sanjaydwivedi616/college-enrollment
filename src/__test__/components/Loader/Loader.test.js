import renderer from 'react-test-renderer';
import Loader from '../../../components/loader/Loader';

describe('When loader page is render', () => {
  it('When Loader is render', () => {
    const loaderText = renderer.create(<Loader />)
    expect(loaderText).toMatchSnapshot();
  });
})
