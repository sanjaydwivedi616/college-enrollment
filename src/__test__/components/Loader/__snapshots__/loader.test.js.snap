// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When loader page is render When Loader is render 1`] = `
<p>
  Loading.....
</p>
`;
